pandas/core/reshape/merge.py

```diff
@@ -2055,6 +2066,23 @@ def _get_single_indexer(
     return libjoin.left_outer_join(left_key, right_key, count, sort=sort)
 
 
+def _get_empty_indexer() -> tuple[npt.NDArray[np.intp], npt.NDArray[np.intp]]:
+    return (
```

Comment:
can you add a short doc-string to these both

Reply:
added docstrings